oath/README.md

```diff
@@ -81,10 +87,13 @@ Authenticators often use QR codes to pass the URL for setting up the credentials
 ```
 
 ### Using the Demo Application <a name="using_demo"></a>
-Run demo app, select "OATH demo" pivot in navigation drawer. Plug in YubiKey and click + FAB button to add credential.  
-To test, use one of the services that provides QR codes and authentication with TOTP as 2nd factor auth.  
-For example, [https://demo.yubico.com/playground](https://demo.yubico.com/playground)  
-Or such services as Facebook, Google, Amazon, Microsoft, etc. All provide 2nd factor authentication with Authenticator app. This demo can be used as such.
+1. Run demo app
+2. Select "OATH demo" pivot in navigation drawer
+3. Plug in YubiKey and click + FAB button to add credential.  
+   To test, use one of the services that provides QR codes and authentication with TOTP as 2nd factor auth.  
+   For example, [https://demo.yubico.com/playground](https://demo.yubico.com/playground)  
+   Or such services as Facebook, Google, Amazon, Microsoft, etc. All provide 2nd factor authentication with Authenticator app. This demo can be used as such.
 
 ## Additional Resources <a name="additional_resources"></a>
-OATH - [YK OATH protocol](https://developers.yubico.com/OATH/YKOATH_Protocol.html)
+* [YKOATH Protocol Specification](https://developers.yubico.com/OATH/YKOATH_Protocol.html)
+* [Yubico Authenticator for Android open source project](https://github.com/Yubico/yubioath-android)
```

Comment:
This authenticator doesn't use YubiKit. So I'm not sure if we actually should mention it here.
Demo shows implementation using YubiKit.

Reply:
Removed. Also checked in some FIDO doc edits